### ISSUE TITLE
Visualization_transform not applied on Insights

### ISSUE THREAD

**Issue by fmartiescofet** [pytorch/captum#855]
## 🐛 Bug

In the class ImageFeature in the file [captum/insights/attr_vis/features.py](https://github.com/pytorch/captum/blob/master/captum/insights/attr_vis/features.py#L118) the visualization_transform is never applied in the visualize function

## To Reproduce

1. Use captum insights with a visualization_transform

## Expected behavior

Should apply the visualization_transform before displaying the original image

## Environment
Describe the environment used for Captum

```

 - Captum / PyTorch Version: 0.4.1/1.10.2
 - OS (e.g., Linux): Linux
 - How you installed Captum / PyTorch (`conda`, `pip`, source): conda
 - Build command you used (if compiling from source):
 - Python version: 3.9.10
 - CUDA/cuDNN version: 11.3.1

**Comment by NarineK** [comment]
cc: @Reubend

**Comment by NarineK** [comment]
@aobo-y is this issue address ? can we close it ?

**Comment by aobo-y** [comment]
@fmartiescofet Thank you for pointing out the bug. It has been fixed in latest v0.5.0
